pep-0001.txt

```diff
@@ -157,21 +157,22 @@ Submitting a PEP
 ----------------
 
 Following a discussion on python-ideas, the workflow varies based on whether
-the PEP author is a core developer. If the PEP author is **not** a
-core developer then the PEP author will need to find a core developer
-*sponsor* for the PEP. The sponsor's job is to provide guidance to the PEP
-author to help them through the logistics of the PEP process (somewhat acting
-like mentor). For the core developer sponsoring, being a sponsor does **not**
-disqualify them from becoming a co-author or BDFL-Delegate later on (but not
-both). The core developer who becomes the sponsor of a PEP is recorded in the
-"Sponsor:" field of the header.
-
-Once a core developer is found that is willing to sponsor the PEP -- whether by
-being an author of the PEP or specifically a sponsor -- and deems the PEP ready
-for submission, the proposal should be submitted as a draft PEP via a
-`GitHub pull request`_.  The draft must be written in PEP style as described
-below, else it will fail review immediately (although minor errors may be
-corrected by the editors).
+the PEP author is a core developer. If the PEP author is **not** a core
```

Comment:
I've always found the language here a but confusing, since the text first states that the workflow depends on whether the author is a core dev or not, and then doesn't say anything about the core dev case.

I propose to change the text to: "[...] the workflow varies based on whether any of the PEP's co-authors are core developers. If one or more of the PEP's co-authors are core developers, they are responsible for following the process outlined below. Otherwise (if none of the co-authors are core developers), then the PEP author(s) will need to find a *sponsor* for the PEP." (And then continue with "Ideally, ...")

Reply:
Nice rewrite, thanks!